modules/apps/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/global.es.js

```diff
@@ -80,7 +80,7 @@ Liferay.Util.PortletURL = {
 	createActionURL,
 	createRenderURL,
 	createResourceURL,
-	createURL
+	createPortletURL
```

Comment:
This should fail sf now, you probably need to sort it

Reply:
I have become too reliant on auto-formatting done by VSCode and Prettier

Reply:
`sort-object-keys` is not an autofixable rule. In any case, you running `yarn format` should be a good sanity check always.